Reference/using-ioc.md

```diff
@@ -130,7 +144,7 @@ As another example, you can do the same with SurfaceControllers. Here we are cre
 
 ##What assemblies and controllers do I need to register?
 
-You need to register all assemblies that may contain MVC or WebApi controllers. In Umbraco this is the `umbraco` assembly which you can get a direct assembly reference to using the example syntax used above:
+You need to register all assemblies that may contain MVC or WebApi controllers. In Umbraco, these include the `umbraco` assembly and assemblies containing treeviews used in the back office (see the examples above). You can get a direct assembly reference with the following syntax:
```

Comment:
Again the mention of 'treeviews' shouldn't exist since that is not an Umbraco term. This can just be reworded as

> You need to register all assemblies that may contain MVC or WebApi controllers this includes both Umbraco and non Umbraco assemblies. For Umbraco, all MVC and WebApi controllers are in the `umbraco` assembly which you can get a direct assembly reference with the following syntax: